src/Process/FeatureProcessing.js

```diff
@@ -34,6 +34,8 @@ export default {
         const zoomDest = extentsDestination[0].zoom;
 
         // check if it's tile level is equal to display level layer.
+        // TO DO updata at all level asked
```

Comment:
can you create an issue please? (or add it to the issue on vector data you already created)

Reply:
It's the first item in my issue on vector tile. I tried to clarified it on my list.